bundles/af-core/src/test/resources/form/password/exporter-password-datalayer.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "password-1c7bc238a6",
+  "fieldType": "password",
+  "name": "password1732094911597",
+  "visible": true,
```

Comment:
Default values should not be present if its not part of JCR

Reply:
default value of visible is false